__mocks__/browser.js

```diff
@@ -0,0 +1,4 @@
+import { setupWorker } from 'msw';
+import handlers from './handlers';
```

Comment:
I suggest we should add handlers file as well in same PR with an example mock because its confusing to see handlers being called without any file

Reply:
Added